wd.html

```diff
@@ -177,21 +177,20 @@
 
 <h3>Firefox</h3>
 
-<p>The driver for Firefox is maintained by the Selenium project
- and works by installing a browser add-on
- in Firefox' profile before starting the browser.
- The WebDriver client then communicates with that add-on
- to instruct the browser what to do.
+<p>Starting with Selenium 3, Mozilla has taken over implementation of Firefox Driver, with <a href=https://github.com/mozilla/geckodriver/releases>Geckodriver</a>, which must be added to your PATH.
+ Gecko Driver works with Firefox 48 and newer, although the implementation is not yet complete, compared to the Firefox Driver built into Selenium 2.
```

Comment:
I would change this to:

> The new driver for Firefox is called _geckodriver_ and works with Firefox 48 and newer. Since the Firefox WebDriver is under development, the newer the Firefox version the better the support.